modules/tinymce/CHANGELOG.md

```diff
@@ -21,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A double bottom border appeared on inline mode editor for the `tinymce-5` skin. #TINY-9108
 - The editor header showed up even with no menubar and toolbar configured. #TINY-8819
 
+### Changed
+- Transparent elements like anchors are now allowed in the root of the editor body. #TINY-9172
+- Transparent elements like anchors can now contain block elements. #TINY-9172
```

Comment:
Changed it to `Transparent elements, like anchors, are now allowed in the root of the editor body if they contain blocks` since that is now the new truth.